src/Microsoft.Diagnostics.Monitoring.Egress/LoggingExtensions.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Logging;
+using System;
+using System.Globalization;
+
+namespace Microsoft.Diagnostics.Monitoring.Egress
+{
+    internal static class EgressOptionsLogging
+    {
+        private const string ProviderOptionLogFormat = "Provider option: {0} = {1}";
+        private const string StreamOptionLogFormat = "Stream option: {0} = {1}";
+
+        public static void LogProviderOption(this ILogger logger, string name, bool value)
+        {
+            logger.LogProviderOption(name, value.ToString(CultureInfo.InvariantCulture));
+        }
+
+        public static void LogProviderOption(this ILogger logger, string name, Uri value)
```

Comment:
>Uri [](start = 79, length = 3)

This will exclude the sas token correct?

Reply:
The AccountUri should not have a SAS token, but you make a good point that we should scrub that out of it (just in case) before it is logged.

Reply:
It not cleans up the account URI before logging, using, etc...